http/auth.go

```diff
@@ -172,11 +174,13 @@ var signupHandler = func(w http.ResponseWriter, r *http.Request, d *data) (int,
 	return http.StatusOK, nil
 }
 
-var renewHandler = withUser(func(w http.ResponseWriter, r *http.Request, d *data) (int, error) {
-	return printToken(w, r, d, d.user)
-})
+func renewHandler(tokenExpireTime time.Duration) handleFunc {
+	return withUser(func(w http.ResponseWriter, r *http.Request, d *data) (int, error) {
+		return printToken(w, r, d, d.user, tokenExpireTime)
+	})
+}
```

Comment:
Please adjust it with the fix in master https://github.com/filebrowser/filebrowser/blob/master/http/auth.go#L175-L178